alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureMonitorRestApiIT.kt

```diff
@@ -48,12 +51,39 @@ import org.opensearch.commons.authuser.User
 import org.opensearch.commons.rest.SecureRestClientBuilder
 import org.opensearch.index.query.QueryBuilders
 import org.opensearch.rest.RestStatus
-import org.opensearch.script.Script
-import org.opensearch.search.aggregations.bucket.composite.CompositeAggregationBuilder
-import org.opensearch.search.aggregations.bucket.composite.TermsValuesSourceBuilder
 import org.opensearch.search.builder.SearchSourceBuilder
 import org.opensearch.test.junit.annotations.TestLogging
 
+val SEARCH_EMAIL_GROUP_DSL = """
+        {
+          "from": 0,
+          "size": 20,
+          "sort": { "email_group.name.keyword": "desc" },
+          "query": {
+            "bool": {
+              "must": {
+                "match_all": {}
+              }
+            }
+          }
+        }
+""".trimIndent()
+
+val SEARCH_EMAIL_ACCOUNT_DSL = """
+                {
+                  "from": 0,
+                  "size": 20,
+                  "sort": { "email_group.name.keyword": "desc" },
+                  "query": {
+                    "bool": {
+                      "must": {
+                        "match_all": {}
+                      }
+                    }
+                  }
+                }
+""".trimIndent()
+
```

Comment:
Move these to their respective `SecureEmailAccountRestApiIT`and `SecureEmailGroupRestApiIT` classes

Reply:
done